tests/Unit/UtilizationMetricsApisTest.php

```diff
@@ -0,0 +1,436 @@
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['alerts'])->toBe($randomRecords);
+});
+
+it('checks the API returns Segments', function () {
+    $randomRecords = random_int(1, 10);
+
+    Segment::factory()->count($randomRecords)->create();
+
+    $softDeleteSegment = Segment::factory()->create();
+    $softDeleteSegment->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['segments'])->toBe($randomRecords);
+});
+
+it('checks the API returns Resource Hub Articles', function () {
+    $randomRecords = random_int(1, 10);
+
+    ResourceHubArticle::factory()->count($randomRecords)->create();
+
+    $softDeleteResourceHubArticle = ResourceHubArticle::factory()->create();
+    $softDeleteResourceHubArticle->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['resource_hub_articles'])->toBe($randomRecords);
+});
+
+it('checks the API returns Events Created', function () {
+    $randomRecords = random_int(1, 10);
+
+    Event::factory()->count($randomRecords)->create();
+
+    $softDeleteEvent = Event::factory()->create();
+    $softDeleteEvent->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['events_created'])->toBe($randomRecords);
+});
+
+it('checks the API returns Forms Created', function () {
+    $randomRecords = random_int(1, 10);
+
+    Form::factory()->count($randomRecords)->create();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['forms_created'])->toBe($randomRecords);
+});
+
+it('checks the API returns Forms Submitted', function () {
+    $randomRecords = random_int(1, 10);
+
+    FormSubmission::factory()->count($randomRecords)->create();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['forms_submitted'])->toBe($randomRecords);
+});
+
+it('checks the API returns Surveys Created', function () {
+    $randomRecords = random_int(1, 10);
+
+    $totalRecords = $randomRecords + (Survey::count());
+
+    Survey::factory()->count($randomRecords)->create();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['surveys_created'])->toBe($totalRecords);
+});
+
+it('checks the API returns Surveys Submitted', function () {
+    SurveySubmission::truncate();
+
+    $randomRecords = random_int(1, 10);
+
+    SurveySubmission::factory()->count($randomRecords)->create();
+
+    $response = get('/api/utilization-metrics');
```

Comment:
We should be using the route name here, not calling the path directly.

Reply:
Done